docs/src/3.4.MouseEvents.mdx

```diff
@@ -75,3 +75,7 @@ Check out below example for automatic/custom hitmap mapping, and regular/instanc
 ## Full Example
 
 <MouseEvents />
+
+## Other
+
+We use `DEFAULT_MOUSE_CLICK_RADIUS` (3 pixels in the screen space) to determine if mouse click event is going to be triggered, specificially if the mouse has moved beyond the `DEFAULT_MOUSE_CLICK_RADIUS` limit, the `onClick` event handler will be canceled, and `onMouseUp` will still get triggered if present. This helps implement certain behaviors such as panning around worldview without firing `onClick` event.
```

Comment:
we don't really need to know that the variable is called `DEFAULT_MOUSE_CLICK_RADIUS`. Either we don't mention anything about this, or we should have a small blurb e.g. "`onClick` events don't fire if the user moves their cursor more than 3 pixels between mouse down and mouse up"

Reply:
Updated: `onClick` events don't fire if the user moves their cursor for more than 3 screen pixels between mouse down and mouse up. This helps implement certain behaviors such as panning around worldview without firing `onClick` event.